picasso-layoutlib-compat/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'com.android.library'
+apply plugin: 'org.jetbrains.kotlin.android'
+apply plugin: 'app.cash.paparazzi'
+apply plugin: 'com.vanniktech.maven.publish'
```

Comment:
I think this should be a jvm module.  From what I can tell, it's only an android module for the purposes of Paparazzi testing?  Solution: make a new module `picasso-paparazzi-sample` and move the PicassoPaparazziTest there.  Eventually, when Paparazzi supports application modules, we can merge that with the currently-existing sample.